src/cmd/create.go

```diff
@@ -505,10 +505,10 @@ func getDBusSystemSocket() (string, error) {
 
 func getEnterCommand(container, release string) string {
 	var enterCommand string
-	containerNamePrefixDefaultWithRelease := utils.ContainerNamePrefixDefault + "-" + release
+	containerNamePrefixDefaultWithRelease := utils.GetDefaultContainerNamePrefix() + "-" + release
 
 	switch container {
-	case utils.ContainerNameDefault:
+	case utils.GetDefaultContainerNamePrefix():
```

Comment:
This doesn't look right. We don't want only the prefix here, but the entire name including the release.